### ISSUE TITLE
Connect results page to firebase API

### ISSUE THREAD

**Issue by unclebinary1001** [kprose07/MEDISCAN#27]
Currently, the results page uses static data. 

The goal is to instead load data on the page from the firebase database. 

Also, there's need to link a click on a medicine card in the home page to launch the results page for the clicked medicine

**Comment by kprose07** [comment]
Yeah so on the narrow down your search page, if a user selects a box we can make a toast that tells the user to please select a medication. Then once user selects medicine we’ll lead them to the results fragment

**Comment by kprose07** [comment]
Then we’ll put the data from the json there in the result fragment file

**Comment by kprose07** [comment]
I wanted to make a collapse all function and expand all but it’s not working out. I’ve made them switchable but I can’t access a book variable from another class kotlin file into a fragment file.  So that’s all that’s left for the results page. Then I’ll start working on the narrow down your search page and attempt to use the hash map etc.

**Comment by kprose07** [comment]
I’ve created an issue for this^
